src/Configuration/ConfigurationFactory.php

```diff
@@ -91,7 +91,7 @@ public function create(?string $path = null, array $paths = []): Configuration
         array_unshift($patchers, new SymfonyParentTraitPatcher());
         array_unshift($patchers, new ComposerPatcher());
 
-        $symbolsConfiguration = $this->configurationWhitelistFactory->createSymbolsConfiguration($config);
+        $symbolsConfiguration = $this->symbolsConfigurationFactory->createSymbolsConfiguration($config);
```

Comment:
should be renamed separately, double check usages of "whitelist" remaining (case insensitive)